backend/wmg/data/transform.py

```diff
@@ -129,6 +130,26 @@ def build_gene_id_label_mapping(gene_ontology_term_ids: List[str]) -> List[dict]
     def build_ontology_term_id_label_mapping(ontology_term_ids: Iterable[str]) -> List[dict]:
         return [{ontology_term_id: ontology_term_label(ontology_term_id)} for ontology_term_id in ontology_term_ids]
 
+    def order_tissues(ontology_term_ids: Iterable[str]) -> Iterable[str]:
+        """
+        Order tissues based on appearance in TissueMapper.HIGH_LEVEL_TISSUES. This will maintain the priority set in
+        that class which is intended to keep most relevant tissues on top and tissues that are related to be placed
+        sequentially
```

Comment:
I understand the bit about keeping relevant tissues on top, but I am not sure what you mean when you say "tissues that are related to be placed sequentially" - is that referring to `ontology_term_ids` after removing high-priority tissues?

Reply:
nope, I'm referring to groups of high-level tissue like:

```
        "UBERON_0001155",  # colon
        "UBERON_0000059",  # large intestine
        "UBERON_0002108",  # small intestine
        "UBERON_0000160",  # intestine
        "UBERON_0000945",  # stomach
```

or 

```
        "UBERON_0003889",  # fallopian tube
        "UBERON_0000995",  # uterus
        "UBERON_0000992",  # ovary
```

or 

```
        "UBERON_0002110",  # gall bladder
        "UBERON_0001255",  # urinary bladder
        "UBERON_0018707",  # bladder organ
```

 and few other groups